### ISSUE TITLE
[Scholomance] Pathing: Lord Alexei Barov - Aggro behaviour

### ISSUE THREAD

**Issue by antonius123** [dalaranwow/dalaran-wow#5355]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Lord Alexei Barov runs through the roof as soon as you aggro him and you need to wait for him to come from the floor above back to the room where you are to fight him. 

**Expected behaviour**: Tell us what should happen instead
Not go through the roof.

**Steps to reproduce the problem**:

1. Aggro Lord Alexei Barov in Scholomance, did the dungeon twice, happened twice.
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
The issue is caused by the huge range of this spell: https://wowgaming.altervista.org/aowow/?spell=17467

I will change the mechanic like that: Lord Alexei Barov will get the aura once he get's infight - not on reset.
